aten/src/ATen/native/native_functions.yaml

```diff
@@ -8807,6 +8807,26 @@
   dispatch:
     CPU, CUDA: linalg_vector_norm_out
 
+- func: linalg_matrix_norm(Tensor self, Scalar? ord=None, int[]? dim=None, bool keepdim=False, *, ScalarType? dtype=None) -> Tensor
+  python_module: linalg
+  dispatch:
+    CPU, CUDA: linalg_matrix_norm
```

Comment:
The implementation doesn't use dispatch stubs and doesn't require explicit custom derivative so the dispatch key should be in this case `CompositeImplicitAutograd` instead of `CPU, CUDA`, or simply omitted, similarly to what is done for `linalg_norm`.
Currently, this function is not supported by autograd and that would be revealed with OpInfo-based tests.

Reply:
Good catch, I also forgot to add OpInfo.